### PR TITLE
typing: adding types for object-map-keys module

### PR DESCRIPTION
**Pull request by Aryaman1706**

### CODE REVIEW

File: packages/object-map-keys/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// Definitions by: Aryaman1706 <https://github.com/Aryaman1706>
+declare function map<T extends object = object>(
+  obj: T,
+  predicate: (value: any, key: string | number, object: T) => string | number
+): object;
+export = map;
```
Comment (PuruVJ) [inline packages/object-map-keys/index.d.ts:6]: Recommend making this `export default map`. There was an issue sometime ago where it had to be converted to export default in lot of packages. @angus-c mind telling what it was 🤔?

**Comment by Aryaman1706** [comment]
@PuruVJ  Thanks a lot for the review.
Made appropriate changes in the latest commit. Kindly have a look.

**Comment by PuruVJ** [comment]
Looks perfect to me

File: packages/object-map-keys/index.d.ts (hunk 1 above)
Comment (angus-c) [reply packages/object-map-keys/index.d.ts]: Yeah it was to make it sync with module package exports

**Comment by angus-c** [comment]
Thank you!

**Pull request merged**